bfasst/opt/base.py

```diff
@@ -1,22 +1,26 @@
 """ Base class for logic optimization tools"""
 import abc
-from bfasst.status import OptStatus
 
 from bfasst.tool import Tool
-from bfasst.status import Status
 from bfasst.utils import print_color
+from bfasst.tool import BfasstException
+
+
+class OptException(BfasstException):
+    """Base class for all exceptions in the opt package"""
 
 
 class OptTool(Tool):
     """Base class for logic optimization tools"""
 
-    success_status = Status(OptStatus.SUCCESS)
-
     def __init__(self, cwd, design, flow_args="") -> None:
         super().__init__(cwd, design)
+        self.yosys_netlist_path = [str(self.design.netlist_path)]
```

Comment:
Why does opt base have a property named *yosys*? That seems incorrect...

Reply:
It is used here in the implementing classes. I'm not the one who wrote this, so if naming is wrong, I don't know why: https://github.com/byuccl/bfasst/blob/e865755c6243aef6ef3b31b6ca143587bcf78bad/bfasst/opt/ic2_lse.py#L47
https://github.com/byuccl/bfasst/blob/e865755c6243aef6ef3b31b6ca143587bcf78bad/bfasst/opt/ic2_synplify.py#L51

Reply:
Ok, don't worry about it then.